lib/get-request-agent.js

```diff
@@ -20,28 +20,29 @@ function getRequestAgent (options) {
   if (agentOptionNames.length === 0) {
     return
   }
-
```

Comment:
nitpick: avoid changes unrelated to the PR :)

Reply:
Oops. I'll remember it. 😃